client/src/shared/components/Iframe/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+interface IframeProps {
+  src: string;
+  className?: string;
+  width?: string;
+  height?: string;
+  title?: string;
+}
+
+const Iframe: React.FunctionComponent<IframeProps> = ({
+  src,
+  className,
+  width,
+  height,
+  title
+}: IframeProps) => {
+  return (
+    <div>
+      <iframe
+        className={`mx-auto m-2 ${className ?? ''}`}
```

Comment:
- no need to add logical operator. since there is already default value.
```suggestion
        className={`mx-auto m-2 ${className}`}
```